README.md

```diff
@@ -335,21 +342,41 @@ Patchwork v2 instance after `Thu,  3 May 2018 14:35:00 +0100` timestamp:
               --restapi https://patchwork.ozlabs.org netdev \
               --lastpatch 'Thu,  3 May 2018 14:35:00 +0100'
 
+<<<<<<< HEAD
```

Comment:
@major , the sktm user guide was expanded so the file needs to be merged properly first :)